src/id.rs

```diff
@@ -111,6 +112,14 @@ impl Debug for PublicId {
     }
 }
 
+impl Deserialize for PublicId {
+    fn deserialize<D: Deserializer>(deserialiser: D) -> Result<Self, D::Error> {
+        let (public_encrypt_key, public_sign_key): (box_::PublicKey, sign::PublicKey) =
+            Deserialize::deserialize(deserialiser)?;
```

Comment:
Does Serde guarantee that this tuple's serialisation will always be identical to the struct's?